.github/workflows/build-ci.yml

```diff
@@ -69,7 +69,7 @@ jobs:
           tags: |
             type=edge,branch=main
             type=raw,priority=950,enable=${{ startsWith(github.ref, 'refs/heads/release/v') }},value=${{ steps.semver.outputs.semver }}
-            type=raw,enable=${{ startsWith(github.ref, 'refs/heads/release/v0.7') }},value=latest
+            type=raw,enable=${{ startsWith(github.ref, 'refs/heads/release/v0.9') }},value=latest
```

Comment:
Some of these are switching from 0.7 to 0.9, while others are switching from 0.8. Is there a reason why? And no way to share a constant using a [matrix](https://docs.github.com/en/actions/learn-github-actions/contexts#matrix-context) or an [inputs](https://docs.github.com/en/actions/learn-github-actions/contexts#inputs-context) context?

Reply:
I think it was due to the previous PR for these straddling a release bump. There may be some better ways to use github action outputs, or scripting, but nothing built into the yaml afaict. https://github.com/deephaven/deephaven-core/issues/1808

Reply:
Oh, maybe inputs would work. Haven't used them before.

Reply:
https://github.com/deephaven/deephaven-core/issues/1842